README.md

```diff
@@ -9,9 +9,9 @@
 
 Hummingbird is a server used to play custom theme songs (or walk-in songs) when a person enters a building. Hummingbird monitors the network and detects when a new device (identified by its MAC address) connects to the router. When a new device connects, Hummingbird plays the song associated with that device.
 
-Hummingbird is designed to run continuously on a server connected to speakers and Wi-Fi. A [Raspberry Pi](http://www.raspberrypi.org/) with a Wi-Fi dongle may prove to be a good setup for Hummingbird.
+Hummingbird has two components: the Django 1.7 server and the Hummingbird daemon. The server is the interface for adding users, device MAC addresses, and songs. The daemon runs in the background sniffing for MAC addresses, and then fires off a local request to the Django server to see if a song should be played, and if so, which one.
```

Comment:
Let's take out the "1.7" part here in the description. Soon, we will probably update the server to 1.9 and it would be easy to forget then to update this. If users are curious about the version, they can check `requirements.txt`.